src/status_im/contexts/shell/share/wallet/view.cljs

```diff
@@ -66,25 +67,65 @@
             :networks            @selected-networks
             :on-share-press      #(share-action qr-url share-title)
             :profile-picture     nil
+            :unblur-on-android?  true
             :full-name           (:name account)
-            :customization-color (:color account)
+            :customization-color  (:color account)
             :emoji               (:emoji account)
             :on-multichain-press #(reset! wallet-type :wallet-multichain)
             :on-legacy-press     #(reset! wallet-type :wallet-legacy)
             :on-settings-press   #(open-preferences @selected-networks)}]]]))))
 
+(def wallet-qr-code-item (memoize wallet-qr-code-item-internal))
+
+(defn- indicator
+  [is-active]
+  [rn/view
+   {:style {:width           8
+            :height          8
+            :borderRadius    4
+            :backgroundColor :white
+            :opacity         (if is-active 1.0 0.5)}}])
+
+(defn- indicator-list
+  [n current-index]
+  [rn/view
+   {:style {:display         :flex
+            :flex-direction  :row
+            :align-items     :center
+            :justify-content :center
+            :gap             8}}
+   (for [i (range n)]
+     ^{:key i} [indicator (= current-index i)])])
+
+(defn render-item
+  [item]
+  (let [width (rf/sub [:dimensions/window-width])]
+    [wallet-qr-code-item {:account item
+                          :index  (:position item)
+                          :width width}]))
+
 (defn wallet-tab
   []
-  (let [accounts (rf/sub [:wallet/accounts])
-        width    (rf/sub [:dimensions/window-width])]
-    [rn/flat-list
-     {:horizontal                true
-      :deceleration-rate         0.9
-      :snap-to-alignment         "start"
-      :snap-to-interval          (- width 30)
-      :disable-interval-momentum true
-      :scroll-event-throttle     64
-      :data                      accounts
-      :directional-lock-enabled  true
-      :render-fn                 (fn [account index]
-                                   (wallet-qr-code-item account width index))}]))
+  (let [accounts      (rf/sub [:wallet/accounts])
+        width         (rf/sub [:dimensions/window-width])
+        current-index (reagent/atom 0)]
+    (fn []
+      [rn/view
+       [rn/flat-list
+        {:horizontal                        true
+         :deceleration-rate                 0.9
+         :snap-to-alignment                 "start"
```

Comment:
does :start work here?

Reply:
it does, this was done as part of this [PR](https://github.com/status-im/status-mobile/pull/18424/files) and ensures that the sides of the previous and next qr-code component can be seen

Reply:
I think @J-Son89 means to try `:start` instead of `"start"` (correct me if I'm wrong, Jamie)

Reply:
oh sorry, I understand now 🙂  have update